bread/layout/components/forms/queryfield.py

```diff
@@ -0,0 +1,201 @@
+import json
+
+import htmlgenerator as hg
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+from djangoql.schema import DjangoQLSchema
+from djangoql.serializers import DjangoQLSchemaSerializer
+
+from bread.layout.components.icon import Icon
+
+
+class BrowseViewSearch(hg.DIV):
```

Comment:
What is the reason that we cannot reuse and adjust the existing Search element? It feels like we add a lot of duplication here.

Reply:
This is because there are 2 input boxes that are switched back and forth. The normal search uses `input[type="text"]` while the advanced search uses `textarea`. There are also different icons for each mode and many different parts that has to directly modify within `Search`, so I decided to make a new one instead but use `Search` as the model component.

Reply:
Okay, I see.